pom.xml

```diff
@@ -8,7 +8,7 @@
   <artifactId>unit-testing</artifactId>
   <version>1.0-SNAPSHOT</version>
 
-  <name>unit-testing</name>
+  <name>unit-testng</name>
   <!-- FIXME change it to the project's website -->
```

Comment:
line 12-13 can be cleaned

Reply:
Thank you, corrected.
Changes will be seen in the following commit.